### ISSUE TITLE
[Billing] Collect billing data for November - Dedicated clusters

### ISSUE THREAD

**Issue by GeorgianaElena** [2i2c-org/infrastructure#5190]
### Instructions for performing this task

Dedicated clusters [instructions](https://infrastructure.2i2c.org/howto/bill/#communities-with-dedicated-cloud-accounts)

### Tasklist

- [ ] Billing data is collected according to instructions.
- [ ] The billing slack was notified about the new billing data collected

**Comment by sgibson91** [comment]
I think I can't start this until at least a couple of days into the new month, right?

**Comment by GeorgianaElena** [comment]
Yes, I noticed the most accurate results (no subsequent changes) are got on the 2nd day of the new month, Dec 2nd would be ideal for the costs to be gathered.
